src/anomalib/utils/metrics/thresholding/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""Thresholding metrics"""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from .base import BaseAnomalyScoreThreshold
```

Comment:
would it be an idea to follow a similar naming convention to `F1AdaptiveThreshold` and `ManualThreshold` and name the base class as `BaseAnomalyScoreThreshold`? @djdameln @ashwinvaidya17 ?